src/py42/modules/archive.py

```diff
@@ -146,4 +146,9 @@ def get_all_org_cold_storage_archives(
             generator: An object that iterates over :class:`py42.response.Py42Response` objects
             that each contain a page of cold storage archive information.
         """
-        return self._archive_client.get_all_org_cold_storage_archives(org_id)
+        return self._archive_client.get_all_org_cold_storage_archives(
```

Comment:
The params did nothing before

Reply:
good find, this was definitely my fault :-/